### PR TITLE
Fix lured Pokémon issues

### LINKED ISSUE #1647: Lured Active but is not Active
Lured not Active Ingame but map sayed yes 

lured

Develop Version actually 

Edit* Timer goes higher by refresh site

Edit2* Now is the Lured Inactive, is this normal?

### PR DESCRIPTION
**Pull request by andrewda**
## Description

This fixes a bug where the Pokémon names are being loaded after lured Pokémon are placed, resulting in them having `undefined` names. Additionally, this fixes a bug where lured Pokémon disappear before the lure expires.

## Motivation and Context

This Pull Requests resolves #1647, which reported the latter of the issues listed above, but also was where I noticed the former.

## How Has This Been Tested?

Tested on Google Chrome/Firefox on Mac OS X and Windows. This shouldn't be a breaking change on any OS/browser, though.

## Screenshots (if appropriate):
(see #1647)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by MonkeyLeeT** [comment]
#1743 ?

**Comment by andrewda** [comment]
@MonkeyLeeT This and #1743 are separate. If I'm not mistaken, #1743 adds a toggle, while this fixes some issues regarding the JSON loading after the lured Pokémon have been loaded.

**Comment by JZ6** [comment]
needs a rebase

**Comment by andrewda** [comment]
@JZ6 I actually have no idea what I just did. Rebasing is not my strong suit...

**Comment by JZ6** [comment]
+1

File: static/map.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ function initMap() {
     google.maps.event.addListenerOnce(map, 'idle', function(){
         updateMap();
     });
-    
+
```
Comment (Chlodochar) [inline static/map.js]: Please remove unrelated changes.

File: static/map.js [hunk 2]
```diff
@@ -543,27 +543,40 @@ function updateMap() {
 
 
          });
+
```
Comment (Chlodochar) [inline static/map.js]: Please remove unrelated changes.
